### PR TITLE
Add nxe1 s0x0x mc

### PR DESCRIPTION
**Pull request by od83**
Hi,

1. I added the Symbols for the Murata PS DC/DC Converter Series NXE1.
The new parts are:
- NXE1S0505MC
- NXE1S0305MC
- NXE1S0303MC

The Datasheet is [here](http://power.murata.com/data/power/ncl/kdc_nxe1.pdf)

I also made a PR for the related [footprint](https://github.com/KiCad/kicad-footprints/pull/2068)

2. I changed the symbols footprint references of the existing NXE2SxxxxMC symbols to the new footprint that I committed for NXE1 and NXE2 series devices.

SYM_NXE1Sxxxx
SYM_NXE2Sxxxx

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the symbol(s) you are contributing
- [x] An example screenshot image is very helpful
- [x] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
  - A new fitting footprint must be submitted if the library does not yet contain one.
- [x] If there are matching footprint PRs, provide link(s) as appropriate
- [x] Check the output of the Travis automated check scripts - fix any errors as required
- [ ] Give a reason behind any intentional library convention rule violation.

### CODE REVIEW

**Comment by cpresser** [comment]
Hi,

is there a reason why you did not made those additions as alias to the existing `NXE2S0505MC`part?
They share the same pinout and package.

**Comment by od83** [comment]
I am new to KiCad and come from an Altium DB with a very different set of rules.
But no, I can not think of a good reason not to use aliases here. Actually, I'd prefer that, too. 
I'll merge NXE1S0x0xMC and NXE2SxxxxMC into NXExSxxxxMC then and do another commit.

**Comment by cpresser** [comment]
> I'll merge NXE1S0x0xMC and NXE2SxxxxMC into NXExSxxxxMC then and do another commit.

You don't need to change the existing symbol. Just add the new variants as aliases to the existing one.
Actually renaming an existing symbol would be a breaking change. That is something that shoule be avoided.

**Comment by od83** [comment]
Oh, yes. I see had another look at the library and understand why NXExSxxxxMC 
is not required. Sorry about that confusion. I'll add the aliases now.

File: Regulator_Switching.dcm [hunk 1]
```diff
@@ -3474,26 +3474,45 @@ K Murata isolated isolation dc-dc converter
...
+#
+$CMP NXE1S0305MC
+D 1W 3.3V to 5V 200mA DC-DC Converter with 3kV isolation
+K isolated isolation dc-dc converter transformer
+F http://power.murata.com/data/power/ncl/kdc_nxe1.pdf
+$ENDCMP
+#
+$CMP NXE1S0505MC
+D 1W 5V to 5V 200mA DC-DC Converter with 3kV isolation
+
```
Comment (cpresser) [inline Regulator_Switching.dcm]: There is some weird empty line here.

Comment (cpresser) [inline Regulator_Switching.dcm]: There is a weird empty line here

Comment (od83) [reply Regulator_Switching.dcm]: weird indeed, I made all changes through the KiCAD symbol editor, not edited the files by hand.
Anyhow, I removed the empty line.
After checking in the symbol editor, I found an odd character at the end of the description line.
Should be fine, now

**Pull request merged**